crypter.go

```diff
@@ -489,7 +496,9 @@ func (obj JSONWebEncryption) Decrypt(decryptionKey interface{}) ([]byte, error)
 		plaintext, err = cipher.decrypt(cek, authData, parts)
 	}
 
-	if plaintext == nil {
+	// BASE64URL(JWE Ciphertext) of an empty byte-string is an empty
+	// byte-string. If obj.ciphertext is empty, plaintext can be nil.
```

Comment:
The semantics of my suggestion here are exactly the same, just reversing the polarity of the words to match the code that follows it.
```suggestion
	// byte-string. If obj.ciphertext was non-empty, empty plaintext means something went wrong.
```